dart/lib/src/protocol/app.dart

```diff
@@ -15,6 +15,16 @@ class App {
     this.deviceAppHash,
   });
 
+  factory App.fromJson(Map<String, dynamic> data) => App(
+        name: data['app_name'],
+        version: data['app_version'],
+        identifier: data['app_identifier'],
+        build: data['app_build'],
+        buildType: data['build_type'],
+        startTime: DateTime.parse(data['app_start_time']),
```

Comment:
out of curiosity for all the setters, if the field doesn't exist in `data` or it's  `null`, what does it happen? I believe it'd break at `DateTime.parse`

Reply:
✅ good catch, thx !